src/MultiSearchPrompt.php

```diff
@@ -43,15 +43,17 @@ public function __construct(
         public mixed $validate = null,
         public string $hint = '',
     ) {
-        $this->trackTypedValue(submit: false, ignore: fn ($key) => Key::oneOf([Key::SPACE, Key::HOME, Key::END, Key::CTRL_A, Key::CTRL_E], $key) && $this->highlighted !== null);
+        $this->trackTypedValue(submit: false, ignore: fn ($key) => Key::oneOf([Key::SPACE, Key::HOME, Key::END, Key::CTRL_A], $key) && $this->highlighted !== null);
 
         $this->initializeScrolling(null);
 
         $this->on('key', fn ($key) => match ($key) {
             Key::UP, Key::UP_ARROW, Key::SHIFT_TAB => $this->highlightPrevious(count($this->matches), true),
             Key::DOWN, Key::DOWN_ARROW, Key::TAB => $this->highlightNext(count($this->matches), true),
-            Key::oneOf([Key::HOME, Key::CTRL_A], $key) => $this->highlighted !== null ? $this->highlight(0) : null,
-            Key::oneOf([Key::END, Key::CTRL_E], $key) => $this->highlighted !== null ? $this->highlight(count($this->matches()) - 1) : null,
+            Key::CTRL_A => $this->highlighted !== null ? $this->toggleAll() : null,
+            Key::CTRL_E => null,
```

Comment:
This probably looks a bit silly...

It's the only way I could find of preventing Prompts from moving to the search input when the user presses `cmd + e` while browsing options. Maybe that's fine, maybe there's a better way?

Reply:
I think it's good. The match statement's `default` branch calls `$this->search()` so that all other key presses will update the search results, which resets the list position. The only way to prevent that for a specific key is to make sure the `default` branch isn't triggered.